README.md

```diff
@@ -169,14 +170,41 @@ If you're using Maven, this is how your `pom.xml` should look like:
 </project>
 ```
 
-With this configutation you can run it from command line:
+With this configuration you can run it from command line:
 
 ```
 $ mvn clean integration-test -Phit-refresh -Dport=8080
 ```
 
 Maven will start the server and you can see it at `http://localhost:8080`.
 
+## Build and Run With Gradle
+
+If you're using Gradle, this is how your `build.gradle` should look like:
+
+```
+plugins {
+    id 'java'
+    id 'application'
+}
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    compile group: 'org.takes', name: 'takes', version: '1.11.3'
+}
+
+mainClassName='App'
```

Comment:
@paulodamaso does this line required to build any app with Takes?

Reply:
@g4s8 It's required to run Takes as the example showed in maven session. I've made some changes to make them identical, please take a look.